code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -430,6 +430,8 @@
 	var/illusion_life_time = 10 SECONDS
 	///How many illusions are created
 	var/illusion_count = 3
+	///If illusions are currently active
+	var/illusions_active = FALSE 
```

Comment:
you can just check the size of the illusions list

Reply:
Implemented